src/main/java/zuve/searchablechests/SearchableChests.java

```diff
@@ -18,7 +18,7 @@
 	public static final Config CONFIG = specPair.getLeft();
 	
 	public SearchableChests() {
-		DistExecutor.runWhenOn(Dist.CLIENT, () -> () -> {
+		DistExecutor.unsafeRunWhenOn(Dist.CLIENT, () -> () -> {
 			ModLoadingContext.get().registerConfig(ModConfig.Type.CLIENT, CONFIG_SPEC);
 			MinecraftForge.EVENT_BUS.register(new ChestEventHandler());
 		});
```

Comment:
I think we can do this to avoid unsafe (I'd push it up but don't have a proper dev environment set up at the moment):
```suggestion
		DistExecutor.safeRunWhenOn(Dist.CLIENT, () -> new SearchableChestsRunnable());
	}
}

class SearchableChestsRunnable implements SafeRunnable {
	private static final long serialVersionUID = 7343468567232854267L;

	@Override
	public void run() {
		ModLoadingContext.get().registerConfig(ModConfig.Type.CLIENT, SearchableChests.CONFIG_SPEC);
		MinecraftForge.EVENT_BUS.register(new ChestEventHandler());
	}
}
```

Reply:
This seems to work!